src/Tizen.NUI/src/public/BaseComponents/ImageView.cs

```diff
@@ -435,8 +436,7 @@ public bool SynchronosLoading
         /// <summary>
         /// Gets or sets whether to synchronous loading the resourceurl of image.<br />
         /// </summary>
```

Comment:
The change itself is ok. I would like to improve the documentation of the property while we are at it. Please describe what operations, methods, behavior is affected by this property. Is it something like I've put in the suggestion below?
```suggestion
        /// Gets or sets whether loading of the image URL is synchronous.<br />
        /// </summary>
        /// <remarks>
        /// Setting this property to true will cause the following operation to be performed synchronously: <see cref="Reload"/>, <see cref="SetImage"/>.
        /// </remarks>
```

Reply:
Thank you I fixed the comment based on the comment you wrote.
How about this?